great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -1178,135 +1178,6 @@ def get_config_with_variables_substituted(
             config = self._project_config
         return DataContextConfig(**self.config_provider.substitute_config(config))
 
-    @public_api
-    def get_batch(  # noqa: PLR0912
-        self, arg1: Any = None, arg2: Any = None, arg3: Any = None, **kwargs
```

Comment:
This has been deprecated within the internal `_get_batch_v3` helper - it acts as a passthrough to `get_batch_list` so we replace all existing v3 uses with that